Marlin/src/lcd/menu/menu_media.cpp

```diff
@@ -126,17 +126,18 @@ void menu_media_filelist() {
   else if (card.isMounted())
     ACTION_ITEM_F(F(LCD_STR_FOLDER " .."), lcd_sd_updir);
 
-  if (ui.should_draw()) for (int16_t i = 0; i < fileCnt; i++) {
-    if (_menuLineNr == _thisItemNr) {
-      card.selectFileByIndexSorted(i);
-      if (card.flag.filenameIsDir)
-        MENU_ITEM(sdfolder, MSG_MEDIA_MENU, card);
+  if (ui.should_draw())
```

Comment:
WDYT of adding curly braces?

Reply:
I didn't dive into this code, just caught sight of strange formatting.